cisco-umbrella/package.yaml

```diff
@@ -137,4 +137,179 @@ pipelines:
       }
       @name = "cisco.umbrella.dns"
       publish "cisco"
+    disabled: true
```

Comment:
Should the data onboarding pipelines themselves generally be disabled? We have it disabled here but not everywhere (e.g. Sophos)